tests/bugs/replicate/bug-921231.t

```diff
@@ -25,7 +25,7 @@ write_to_file &
 write_to_file &
 wait
 #Test if the MAX [F]INODELK fop latency is of the order of seconds.
-inodelk_max_latency=$($CLI volume profile $V0 info | grep INODELK | awk 'BEGIN {max = 0} {if ($6 > max) max=$6;} END {print max}' | cut -d. -f 1 | egrep "[0-9]{7,}")
+inodelk_max_latency=$($CLI volume profile $V0 info | grep INODELK | awk 'BEGIN {max = 0} {if ($6 > max) max=$6;} END {print max}' | cut -d. -f 1 | egrep "[0-9]{9,}")
 TEST [ -z $inodelk_max_latency ]
 
```

Comment:
Could you share the values of $max with and without your patch? Trying to understand the number manipulation here as the test always passes for me as it is.

Reply:
Even i am not able to reproduce in softserve environment, it is failing on the regression server only.On regression server the value is showing
17:00:54 not ok 11 [ 3650/ 1] < 29> '[ -z 2796547 ]' -> ''
the test case is expecting value should be NULL

Reply:
Maybe we could temporarily add a test in the .T to display the `max` value. What do you think?

Reply:
the test case expects inodelk latency should be less than 1 sec so i have updated the test case. Earlier we print latency in microsec that's why test case used to ignore 7 digits now we are using nanosec so we have to ignore 10 digits.